app/src/main/res/values/strings.xml

```diff
@@ -50,4 +50,5 @@
     <string name="title_companion_dialog">Companion Dialog</string>
     <string name="mime_type_text">text/plain</string>
```

Comment:
I wouldn't have expected `text/plain` to be in the resource strings. I don't think we plan to localize this string, and having it here in the resource could lead to an accidental translation.